woocommerce/woocommerce
# Product Collection: Implement the Hand-Picked collection

### Changes proposed in this Pull Request:





Closes #50862

This PR adds the Hand-Picked collection to the Product Collection block.

| Picker | Initial (empty) state | w/ picked products | w/ On Sale filter |
| ------------- | ------------- | ------------- | ------------- |
| <img width="648" alt="Screenshot 2024-09-13 at 14 21 45" src="https://github.com/user-attachments/assets/194495e2-8fe4-4989-864e-0661a98b65fb"> | ![image](https://github.com/user-attachments/assets/eb55b6fe-6e6c-4806-a4c5-541ea4ec5d40) | <img width="650" alt="Screenshot 2024-09-13 at 14 23 09" src="https://github.com/user-attachments/assets/499026c9-4d57-4ef6-85ac-fc20d6c3c443"> | <img width="651" alt="Screenshot 2024-09-13 at 14 23 22" src="https://github.com/user-attachments/assets/5095c545-c16a-41d1-980d-cacfa0a1a38e"> |



### How to test the changes in this Pull Request:

> [!IMPORTANT]
> See [Product Collection: Update heading for Hand-Picked collection](https://github.com/woocommerce/woocommerce/pull/51643#top) for updated testing instructions.



- Create a new post.
- Add the Product Collection block and select the Hand-Picked collection.
- Ensure the initial/empty state of the block displays `No products chosen yet. Use "Hand-Picked Products" in Inspector Controls to choose products to display`.
- Ensure the hand-picked control is visible at the top of the block settings panel (see image above).
- Ensure the hand-picked control is unavailable via the `Filters` setting dropdown, when no products are selected.
	- Save the draft and ensure the empty block is not rendered in the front end.
- Add some products in random order via the hand-picked control.
	- Ensure the "as entered" order is preserved in both editor and front end.
- Apply other filters (e.g., on sale) and ensure the results are as expected.



### Changelog entry







<details>
<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Product Collection: Implement the Hand-Picked collection
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
